controller/system_rollout_controller.go

```diff
@@ -457,11 +459,8 @@ func (c *SystemRolloutController) systemRollout() error {
 		}
 		wg.Wait()
 
-		// Need to wait until backing images are restored, so the volumes with backingimages won't be rejected by webhook when restoring.
-		// PV/PVC restoration depends on Volume, so move them after volume restoration.
+		// Need to wait until backingimages are restored, so the volumes with backingimages can be restored.
 		c.restore(types.LonghornKindVolumeList, c.restoreVolumes, log)
-		c.restore(types.KubernetesKindPersistentVolumeList, c.restorePersistentVolumes, log)
-		c.restore(types.KubernetesKindPersistentVolumeClaimList, c.restorePersistentVolumeClaims, log)
```

Comment:
Any reason that we are touching this logic in this PR?

Reply:
Good catch. Looks like a consequence of a bad rebase on https://github.com/longhorn/longhorn-manager/pull/2819. Fixing.